### DISCUSSION TITLE
how can I add an image to a page

### DISCUSSION THREAD

**Discussion by iratheslayer** [colinwilson/lotusdocs#117] [Feature Requests]
I couldn't find any documentation on adding images.
Adding via HTML tags doesn't work

**Comment by Nadine2016** [comment]
you can use markdown to add images `![My image](/img/my-image.svg)`

**Reply by iratheslayer** [reply]
it doesnt work:(
can  you add ur example from IDE? 
and what it looks like in the demo

**Reply by artberger** [reply]
Where is your image located? We have them in `assets/img`

**Reply by iratheslayer** [reply]
i put them with md file together
these images are illustrations of my file, in the previous theme i put them together in one folder
